shared/src/commonMain/kotlin/com/example/myapplication/shared/root/PreviewRootComponent.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.myapplication.shared.root
+
+import com.arkivanov.decompose.router.stack.ChildStack
+import com.arkivanov.decompose.router.stack.StackNavigation
+import com.arkivanov.decompose.value.MutableValue
+import com.arkivanov.decompose.value.Value
+import com.example.myapplication.shared.main.PreviewMainComponent
+import com.example.myapplication.shared.root.RootComponent.Child
+
+object PreviewRootComponent : RootComponent {
+
+    private val navigation = StackNavigation<Unit>()
+
+    override val stack: Value<ChildStack<*, Child>> =
+        MutableValue(
+            ChildStack(
+                configuration = navigation,
```

Comment:
We don't need `StackNavigation` here. We can just pass `Unit`.

Reply:
Thank you but `RootPreview` has been removed.